Adds Various Classic Cocktails

RusselNotSCP opened the pull request:
## About The Pull Request
TL;DR: "What do you mean we didn't have an old fashioned in the game before?"

This adds a bunch of different classic cocktails to the game that I always felt were rather odd holes in the lineup of drinks that bartenders could make, as well as a few less classic ones 'cause why not. Adds thirteen new alcoholic drinks and one non alcoholic one. Mechanically relevant drinks include the Blue Blazer and Hot Toddy which warm you up like coffee, Bitters and Soda which settles upset stomachs, and the Suffering Bastard which, in it's role as a hangover cure, calms nausea, prevents headaches, and very slowly heals brain damage, giving a way to heal chip brain damage that doesn't require chemistry or surgery.

Full list here: https://hackmd.io/@0lHAWBNkSXixU4v6xxYS_w/SkjlmIbsgx

<img width="439" height="435" alt="drinks" src="https://github.com/user-attachments/assets/406ca285-63af-49d4-b0e8-176edb251291" />


PS This is basically my first real project and also my first time spriting for anything, so I would really appreciate any advice or reccomendations.
## Why It's Good For The Game
More variety for service is always nice, but also many of these drinks are extremely commonly ordered in real world bars. Adding them here fills some weird gaps in the bar's line up, allows people to order more drinks that they personally like, and decreases the amount of awkward interactions in which people order a drink that they're familiar with and then get confused when a partender somehow can't give them an daiquiri despite having everything to make it.
## Changelog
:cl:
add: You can now make 15 new classic (and some less classic) cocktails. Includes old standards like the Old Fashioned, Daiquiri, Ramos Gin Fizz, and more!
/:cl:

RusselNotSCP committed:
Restored from backup

I suck at using git.

RusselNotSCP committed:
Reworked Old Fashioned desc

Simplified it a lot. Will restore some of the more detailed info if I ever end up doing an Improved Whiskey Cocktail.

RusselNotSCP committed:
Added Flip

Added flip cocktail and sprite (which maybe could use a bit more work, but it's servicable.) Also decreased the boozepwr on the blue blazer to more accurately reflect the fact that it's half water.

RusselNotSCP committed:
Changed Sangria Sprite

Shrunk and simplified.

RusselNotSCP committed:
Merge branch 'tgstation:master' into Classic-Cocktails

RusselNotSCP committed:
Adds Suffering Bastard Headache supression

This includes adding a status effect and trait for soothing headaches in the same way as menthol does. As far as I can tell the only thing that can actually give you headaches are diseases, but this supresses them there so job done I guess.

RusselNotSCP committed:
Fixed Blue Blazer Crafting Recipie

No longer duplicates your glass for no reason. Icon is still wrong and the crafting time is weirdly slow despite me changing it...

RusselNotSCP committed:
Blue Blazer converted to normal reaction recipe

Removed old tablecraft recipie since I wasn't able to do with it what I wanted and just turned blue blazer into a drink made the normal way by mixing and heating. Moved old stuff to a different branch if I ever want to try my fire-setting drink ideas again.

RusselNotSCP committed:
Merge branch 'tgstation:master' into Classic-Cocktails

RusselNotSCP committed:
Update drinkingglass.dm

RusselNotSCP committed:
fixed minor typo

begone comma

RusselNotSCP committed:
Added missing trailing newline

oops

RusselNotSCP committed:
Update drinks_alcoholic.dm

ExcessiveUseOfCobblestone commented on code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm:
```diff
@@ -2894,6 +2894,147 @@
+	. = ..()
+	affected_mob.apply_status_effect(/datum/status_effect/headache_soothed) //prevents headaches
+	affected_mob.adjust_disgust(-5 * REM * seconds_per_tick) //removes disgust, same with sol dry
+	if(affected_mob.adjustOrganLoss(ORGAN_SLOT_BRAIN, -0.5 * REM * seconds_per_tick * normalise_creation_purity(), required_organ_flag = affected_organ_flags)) //heals brain damage very slowly, about 12 damage per 5u
```
This is a bar drink, organ damage really ought not to be easy to heal with chems let alone bar drinks.

Needs to be extremely limited if you really want it in, otherwise should just drop this line.

RusselNotSCP replied:
My idea with this was that it was mostly a way for medbay to offload the fixing of minor brain damage onto service in the same way that they're able to offload every other kind of organ damage onto the chef, but the amount that it repairs is so little that you'd end up damaging your liver before you fixed a large amount of brain damage that medbay might want to fix instead of just largely ignoring. Also, I'm not entirely sure what you mean when you say that organ damage shouldn't be easy to fix with chems, when mannitol is better than this, provided roundstart, and one of the few chems in the game that's ghetto chem viable. If you still think that's too much, though, I'll remove it.

ExcessiveUseOfCobblestone replied:
im stupid

ExcessiveUseOfCobblestone commented:
I generally am not a fan of more drinks because I feel like a lot of them just see zero use if they have no functionality, aren't associated with a bounty, or part of the serving robot menu.

Ideally any new drinks for me to sub to the PR needs to at minimum hit one of those.

RusselNotSCP committed:
Add bounty and restaurant stuff

Added most drinks to the bounty pool, and a few that I felt were fitting to the customer robot's menu.

RusselNotSCP committed:
Makes flip make you do a flip

Seemed like an obvious joke to make. Kept the probability low so that people get surprised by it potentially, rather than just being completely obvious.

RusselNotSCP committed:
Merge branch 'master' into Classic-Cocktails

RusselNotSCP committed:
Merge branch 'master' into Classic-Cocktails

deathrobotpunch commented:
still no lemon lime bitters :(

deathrobotpunch commented:
> Sorry, I'm not australian.

You should try it. it’s lemonade (Kirk’s works best but I don’t know the american equivalent that isn’t sprite), lime cordial and a small amount of angostura bitters
